<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../calculator.png</normaloff>../calculator.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 56, 70);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>526</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>526</width>
             <height>164</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="Arithmetic">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(94, 92, 100);</string>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>526</width>
             <height>163</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="Answer">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(94, 92, 100);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="ButtonMPlus">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(192, 191, 188);</string>
              </property>
              <property name="text">
               <string>M+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ButtonMMinus">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(192, 191, 188);</string>
              </property>
              <property name="text">
               <string>M-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ButtonMR">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(192, 191, 188);</string>
              </property>
              <property name="text">
               <string>MR</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ButtonMC">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(192, 191, 188);</string>
              </property>
              <property name="text">
               <string>MC</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ButtonDel">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(154, 153, 150);</string>
              </property>
              <property name="text">
               <string>Del</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="Button7">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(94, 92, 100);
color: rgb(222, 221, 218);</string>
              </property>
              <property name="text">
               <string>7</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Button8">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(94, 92, 100);
color: rgb(222, 221, 218);</string>
              </property>
              <property name="text">
               <string>8</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Button9">
              <property name="toolTipDuration">
               <number>-7</number>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(94, 92, 100);
color: rgb(222, 221, 218);</string>
              </property>
              <property name="text">
               <string>9</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ButtonDiv">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(198, 70, 0);
color: rgb(0, 0, 0);</string>
              </property>
              <property name="text">
               <string>/</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ButtonClear">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(154, 153, 150);</string>
              </property>
              <property name="text">
               <string>C</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="Button4">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(94, 92, 100);
color: rgb(222, 221, 218);</string>
              </property>
              <property name="text">
               <string>4</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Button5">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(94, 92, 100);
color: rgb(222, 221, 218);</string>
              </property>
              <property name="text">
               <string>5</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Button6">
              <property name="toolTipDuration">
               <number>-7</number>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(94, 92, 100);
color: rgb(222, 221, 218);</string>
              </property>
              <property name="text">
               <string>6</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ButtonMulti">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(198, 70, 0);
color: rgb(0, 0, 0);</string>
              </property>
              <property name="text">
               <string>*</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ButtonLeft">
              <property name="styleSheet">
               <string notr="true">color: rgb(222, 221, 218);</string>
              </property>
              <property name="text">
               <string>(</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="Button1">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(94, 92, 100);
color: rgb(222, 221, 218);</string>
              </property>
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Button2">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(94, 92, 100);
color: rgb(222, 221, 218);</string>
              </property>
              <property name="text">
               <string>2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Button3">
              <property name="toolTipDuration">
               <number>-7</number>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(94, 92, 100);
color: rgb(222, 221, 218);</string>
              </property>
              <property name="text">
               <string>3</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ButtonMinus">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(198, 70, 0);
color: rgb(0, 0, 0);</string>
              </property>
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ButtonRight">
              <property name="styleSheet">
               <string notr="true">color: rgb(222, 221, 218);</string>
              </property>
              <property name="text">
               <string>)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="Button0">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(94, 92, 100);
color: rgb(222, 221, 218);</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ButtonPoint">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(94, 92, 100);
color: rgb(222, 221, 218);</string>
              </property>
              <property name="text">
               <string>,</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ButtonIs">
              <property name="toolTipDuration">
               <number>-7</number>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(222, 221, 218);</string>
              </property>
              <property name="text">
               <string>=</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ButtonPlus">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(198, 70, 0);
color: rgb(0, 0, 0);</string>
              </property>
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(198, 70, 0);
color: rgb(0, 0, 0);</string>
              </property>
              <property name="text">
               <string>1/x</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
